website/content/en/highlights/2021-07-21-0-16-upgrade-guide.md

```diff
@@ -15,10 +15,7 @@ Vector's 0.16.0 release includes three **breaking changes**:
 1. [Component name field renamed to ID](#name-to-id)
 1. [Datadog Log sink encoding option removed](#encoding)
 1. [Renaming of `memory_use_bytes` internal metric](#memory_use_bytes)
-
-And one **deprecation**:
-
-1. [Vector source/sink version 2 released](#vector_source_sink)
+1. [`datadog_logs` source renamed to `datadog_agent`](#datadog_logs_rename)
```

Comment:
This was previously a separate highlight file, but I moved it into here since it is a breaking change.